db-helpers.js

```diff
@@ -1,11 +1,41 @@
 const pg = require('pg')
 
+function setTimestampParsersToReturnStrings(types) {
+    // 1114 and 1184 are the OID for postgres TIMESTAMPTZ and TIMESTAMP
+    // This tells node-pg not to convert these to Js Date objects, just return the string value from postgres
+    [1114, 1184].forEach( (oid) => {
+        types.setTypeParser(oid, (stringValue) => {
+            return stringValue
+        })
+    })
+}
+
+// Wraps pg.connect so that we can change some default behavior for node-pg
+function openDbConnection (callback) {
+    // override default behavior for node-pg here:
+    setTimestampParsersToReturnStrings(pg.types)
+
+    // open the database connection and execute query/callback
+    pg.connect(process.env.DATABASE_URL, function (err, client, done) {
+        if (err) {
+            console.log("Database connection error: ", err)
+        } else {
+            if (callback) {
+                callback(err, client, done)
+            }
+        }
+    })
+}
+
+
 function createUsersTable (callback) {
     const query = `CREATE TABLE IF NOT EXISTS users(
         id SERIAL PRIMARY KEY NOT NULL,
-        email TEXT UNIQUE NOT NULL
+        email TEXT NOT NULL,
+        date_submit TIMESTAMP WITH TIME ZONE NOT NULL,
+        unique (email, date_submit)
```

Comment:
I changed the unique constraints to account for a case where someone fills out the form, but doesn't choose to join, then fills out the form again weeks or months later. We still want to invite the person when they fill out the form again.

This means that the `users` table is now more like a `form_submissions` table and should be renamed at some point soon after this change is reviewed and merged.